modules/virt-setting-policy-attributes.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-schedule-vms.adoc
+
+[id="virt-setting-policy-attributes_{context}"]
+= Setting a policy attribute and CPU feature
+
+You can set a policy attribute and CPU feature for each virtual machine (VM) to ensure that it is scheduled on a node according to policy and feature. The CPU feature that you set is verified to ensure that it is supported by the host CPU or emulated by the hypervisor.
+
+.Procedure
+
+. Edit the `domain` spec of your VM configuration file. The following example sets the CPU feature and the `require` policy for a virtual machine instance (VMI):
```

Comment:
The modular documentation guidelines suggest using an unnumbered bullet instead of an ordered list for single-step procedures. 
Applies to the other two procedures as well.

Reply:
Fixed.